GVFS/GVFS.Common/GVFSContext.cs

```diff
@@ -28,6 +28,7 @@ public GVFSContext(ITracer tracer, PhysicalFileSystem fileSystem, GitRepo reposi
         public void Dispose()
         {
             this.Dispose(true);
+            GC.SuppressFinalize(this);

```

Comment:
Technically, I _think_ this is not necessary (and is a no-op), as this method does not have a finalizer, and hence no finalizer to suppress.

Reply:
Yeah but to keep it consistent with the suggested implementation and if by some tiny chance a finalizer is added we won't have to remember to add this.